docs/djdknativecrypto.md

```diff
@@ -38,12 +38,12 @@ This option controls the use of OpenSSL native cryptographic support.
 
 ## Explanation
 
-OpenSSL support is enabled by default for the Digest, CBC, GCM, and RSA algorithms. If you want to turn off the OpenSSL implementation, set this option to `false`.
+OpenSSL support is enabled by default for the <!--Digest,--> CBC, GCM, and RSA algorithms. If you want to turn off the OpenSSL implementation, set this option to `false`.
 
-If you want to turn off the Digest, CBC, GCM, or RSA algorithms individually, use the following system properties:
+If you want to turn off the an algorithms individually, use the following system properties:
 
 - [`-Djdk.nativeCBC`](djdknativecbc.md)
-- [`-Djdk.nativeDigest`](djdknativedigest.md)
+<!--- [`-Djdk.nativeDigest`](djdknativedigest.md)-->
```

Comment:
This causes a hiccup in the bulletted list.

Reply:
Yep, moved out now.